src/utils/createCoreModule/utils.ts

```diff
@@ -28,6 +28,7 @@ export const CoreModuleNames: CoreModuleName[] = [
   'farmer',
   'milk',
   'delivery',
+  'memo',
```

Comment:
so make sure when you merge my stuff in with yours, you add the memo core module to the list of permitted apis for monitors and admins or else memo data won't get loaded in for them. it will be in this file too

Reply:
I'll wait for yours to go in :P